app/assets/stylesheets/modern-business.scss

```diff
@@ -241,3 +247,33 @@ footer {
   font-size: 40px;
   text-align: center;
 }
+
+/* Countdown Clock */
+
+#clockdiv{
+  font-family: sans-serif;
+  color: #fff;
+  display: inline-block;
+  font-weight: 50;
+  text-align: center;
+  font-size: 30px;
+}
+
+#clockdiv > div{
+  padding: 10px;
+  border-radius: 2px;
+  background: #044A8E;
+  display: inline-block;
+}
+
+#clockdiv div > span{
+  padding: 15px;
+  border-radius: 3px;
+  background: #317EAC;
```

Comment:
Color literals like `#317EAC` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#317EAC` should be written as `#317eac`